src/service/iot_config.proto

```diff
@@ -346,14 +353,31 @@ message gateway_load_region_req_v1 {
   bytes signature = 4;
 }
 
-message gateway_load_region_res_v1 {}
+message admin_load_region_res_v1 {}
 
-message gateway_location_req_v1 {
-  bytes gateway = 1;
+message admin_add_key_req_v1 {
+  enum key_type_v1 {
+    // administrator key
+    operator = 0;
```

Comment:
Can we call this `administrator`? 

`operator` has been used in the past to mean many things other than `admin`.

Reply:
sure thing

Reply:
[89b4930](https://github.com/helium/proto/pull/288/commits/89b493096d25504180a737e8304ff011a8ce50cb)